ee/observe/explorer_handler.go

```
@@ -0,0 +1,551 @@
+)
+
+var (
+	errInvalidObserveRange    = errors.New("invalid Observe time range")
+	errInvalidObservePlatform = errors.New("invalid Observe platform")
+	errInvalidObserveFilter   = errors.New("invalid Observe filter")
+	errInvalidIdentityFilter  = errors.New("invalid Identity filter")
+	// The identity cohort behind an attribute filter is copied onto the wire
+	// as an external table on every request, so it has a hard cap. Past it the
+	// only honest answers are "narrow the filter" or a silently partial
+	// result; this is the first one.
+	errObserveCohortTooLarge = errors.New("Observe identity cohort too large")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)